packages/mui-material/src/CardHeader/CardHeader.d.ts

```diff
@@ -5,62 +5,71 @@ import { OverridableComponent, OverrideProps } from '../OverridableComponent';
 import { Theme } from '..';
 import { CardHeaderClasses } from './cardHeaderClasses';
 
+export interface CardHeaderOwnProps<
+  SubheaderTypographyComponent extends React.ElementType = 'span',
+  TitleTypographyComponent extends React.ElementType = 'span',
```

Comment:
Could you keep the same order of parameters as in the type map?

Reply:
Fixed